plugins/fritz-box7490/de/fritzbox7490_

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+# Munin plugin um die aktuellen Werte der Datei fritzbox7490.log auszuwerten. 
+# internet connection by reading the top_status.htm from the 
+# Don't forget zu fill in the following lines into the munin-node
+#
+# - normally at /etc/muni/plugin-conf.d/ - an than restart munin
+#
+#   [fritzbox_]
+#   user root
+#
+# LLOSS: Leitungsdämpfung   = line loss
+# DRATE: Datenrate          = data rate
+# SNR:   Störabstand        = signal-to-noise ratio
+# DATAV: Datenvolumen       = data transfer
+# ES:    Sek. m. Fehlern    = sec. w. errors
+# SES:   Sek. m. v. Fehlern = sec w. m. errors
+# VERB:  Anzahl tägl. Verb. = count of daily connections
+# UPT:   Zt. seit Reboot    = router uptime
+#                                     Jo Hartmann (Version 16.0126)
+#
+
+###################################
+## Personal config Section Begin ##
+################################### 
+
+logfile=/var/log/fritzbox7490.log
+
+###################################
+## Personal config section End   ##
+###################################
+
+function Echo() {
+# Munin hat probleme mit Sonderzeichen ÄÖÜ äöüß
+# Munin has problems with special characters ÄÖÜ ääüß 
+  konvert=`echo $* | sed s/Ã„/Ae/g | sed s/Ã–/Oe/g | sed s/Ãœ/Oe/g | sed s/Ã¤/ae/g | sed s/Ã¶/ue/g | sed s/Ã¼/ue/g | sed s/ÃŸ/ss/g` 
+  echo $konvert
+}
+
+# Wildcard-Text erkennen 
+# Detect wildcard text
+  id=$0
+  id=${id##*_}
+
+# Prüfen ob die Werte-Datei existiert, wenn ja einlesen sonst Abbruch
+# Check if the values ??file exists, if yes read in otherwise abort
+  if [ -f $logfile ]; then
+       werte=$(cat $logfile)
+  else
+       echo Die Datei $logfile wurde nicht gefunden! Abbruch! >&2
+       exit 1
+  fi
+
+
+# Standard Config Section Begin ##
+  if [ "$1" = "autoconf" ]; then
```

Comment:
"autoconf" is meant to check, if all requirements for this plugin are satisfied. Here you could check if the login works or if curl is available.